test/Microsoft.Azure.SignalR.Tests/Microsoft.Azure.SignalR.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net7.0</TargetFramework>
```

Comment:
hm.. I think we might need to separate net7 support to another  PR or maybe in the same PR supporting client invocation, we need to make sure net7 features like client invocation are supported before we claim that the package now supports net7

Reply:
OK.